docker-compose.yml

```diff
@@ -57,11 +55,11 @@ services:
       - S3_ENDPOINT=${S3_ENDPOINT}
       - S3_ACCESS_KEY=${S3_ACCESS_KEY}
       - S3_SECRET_KEY=${S3_SECRET_KEY}
+      - LOCAL_ROOT=/data
       - APP_COMPONENT=${APP_COMPONENT:-api}
     volumes:
       - .:/app
-      - ${RGDPS_LEVELS_DIR}:/data/levels
-      - ${RGDPS_SAVES_DIR}:/data/saves
+      - ${RGDPS_DIR}:/data
```

Comment:
previous comment really shines here, someone could change one and not the other.. oops!